packages/components/src/Filter/_docs/Filter.stories.tsx

```diff
@@ -5,7 +5,8 @@ import { FilterButton, FilterButtonRemovable } from "~components/FilterButton"
 import { Filter, FilterContents } from ".."
 
 export default {
-  title: "Components/Filter",
+  tags: ["autodocs"],
```

Comment:
You need to remove the `tags: ["autodocs"]` from the updated Filter docs (the ones I added `mdx` files for)